src/static/libs/js/views/player-page/index.js

```diff
@@ -0,0 +1,119 @@
+require('../../navbar.js');
+require('../../auth.js');
+require('../../mediaPlayer.js');
+require('../../clipCreatedModal.js');
+require('../../createPlaylistModal.js');
```

Comment:
They do indeed smell weird...I required these .js files because each calls some JS within them on load (for example to add event listeners).

I'm sure these can be cleaned up so we don't have to require the whole file to achieve the same result, but I was going to skip that clean up for now if it's not too much trouble...